docs/source/API/core/Numerics.rst

```diff
@@ -24,3 +24,11 @@ library mathematical functions, such as ``fabs``, ``sqrt``, and ``sin``.
 
 The header ``<Kokkos_NumericTraits.hpp>`` implements a new facility that is being added to the C++23 standard library and
 is intended as a replacement for ``std::numeric_limits``.
+
+.. toctree::
+   :maxdepth: 1
+
+   numerics/bit-manipulation.md
```

Comment:
Shouldn't this be `numerics/bit-manipulation.rst`?

Reply:
That is a good point.  I copy/pasted from the same file without paying attention.  Want me to fix here or ok to deal with it as a follow up?

Reply:
Ah looks like the links in `Numerics.rst` haven't been replaced along with the md to rst file conversions then. I think it's fine to fix those as a follow up.